src/bootstrap/README.md

```diff
@@ -111,6 +111,42 @@ compiler. What actually happens when you invoke rustbuild is:
 The goal of each stage is to (a) leverage Cargo as much as possible and failing
 that (b) leverage Rust as much as possible!
 
+## Incremental builds
+
+You can configure rustbuild to use incremental compilation. Because
+incremental is new and evolving rapidly, if you want to use it, you
+must replace the snapshot with a locally installed nightly build of
+rustc. You will want to keep this up to date.
+
+So, first thing is to install a nightly, presumably using
+`rustup`. You will then want to configure your directory to use this
+build, like so:
+
+```
+# you don't have to use a build directory, I just think it's nicer --nmatsakis
+> mkdir build
+> cd build
+
+# ok, the --enable-debug and --enable-optimize are not strictly needed,
+# but I recommend them personally. --nmatsakis
+> ../configure --enable-rustbuild --local-rust-root=~/.cargo/ --enable-local-rust \
```

Comment:
I *think* that `--local-rust-root` and `--enable-local-rust` can both be elided here. At least rustbuild only takes a look at `--enable-local-rust`, so the other two configuration options are ignored by rustbuild.